packages/manager/freefax/src/notifications/freefax-notifications.factory.js

```diff
@@ -1,5 +1,10 @@
 import angular from 'angular';
-import _ from 'lodash';
+
+import extend from 'lodash/extend';
```

Comment:
Please use `assignIn` instead

https://github.com/lodash/lodash/wiki/Changelog#compatibility-warnings

Reply:
done

@FredericEspiau thx for reviewing ;)